scripts/Dockerfile-sync-check

```diff
@@ -1,12 +1,28 @@
 FROM fedora:36
+
 LABEL org.opencontainers.image.source https://github.com/chainsafe/forest-iac
 
 # Install dependencies
-RUN dnf install -y docker docker-compose ruby ruby-devel make gcc
-RUN gem install docker-api slack-ruby-client sys-filesystem
+RUN dnf -y install \
+    docker \
+    docker-compose \
+    ruby \
+    ruby-devel \
+    make \
+    gcc \
+    s3fs-fuse \
+    && dnf clean all
 
-WORKDIR /chainsafe
+# Gem install dependencies
+RUN gem install \
+    docker-api \
+    slack-ruby-client \
+    activesupport
 
+# Set a working directory and copy files
+WORKDIR /usr/src/app
 COPY ruby_common ruby_common
-COPY sync_check/* ./
+COPY daily_snapshot/*.rb daily_snapshot/upload_snapshot.sh daily_snapshot/run_service.sh ./
```

Comment:
This does not look right. You copy the daily snapshot files instead of the sync check files? That appears to be a mistake.

Reply:
Yeah sure - I just corrected it.